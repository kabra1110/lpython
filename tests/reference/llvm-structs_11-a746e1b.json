{
    "basename": "llvm-structs_11-a746e1b",
    "cmd": "lpython --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/errors/structs_11.py",
    "infile_hash": "9cb6c80ad837ba66472a91b22e9068ec439b6a2a179a452d90d84c78",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "llvm-structs_11-a746e1b.stderr",
    "stderr_hash": "58e383d2ac915263088426f1b511760a8cb9ef3dd6f24cb207eda4de",
    "returncode": 3
}