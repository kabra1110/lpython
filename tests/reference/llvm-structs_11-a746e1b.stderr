code generation error: Printing support is not available for `Foo` type.
  --> tests/errors/structs_11.py:16:1
   |
16 | print(bar)
   | ^^^^^^^^^^ 
